CHANGES.rst

```diff
@@ -17,6 +17,11 @@ Unreleased
   than 500 KiB, even when the parameters did not exceed their respective permitted
   maximum lengths.
 
+**Deprecated**
+
+- Legacy modmail is slated for deprecation by Reddit in June 2021. See
+  https://redd.it/mar9ha for more info.
```

Comment:
Can we use the full link rather than the shortlink here and in the other locations?